Adyen/Model/PrefilledShopperInformation.swift

```diff
@@ -39,6 +64,17 @@ public struct PrefilledShopperInformation: ShopperInformation {
     ///   - deliveryAddress: The delivery address of the shopper, optional.
     ///   - socialSecurityNumber: The social security number of the shopper, optional.
     ///   - card: Shopper's card basic information, optional.
+    @available(*, deprecated, message:
```

Comment:
@neelSharma12 would you mind adding the required changes to the `MIGRATION.md`? :)

Reply:
87d6b7317958c614da3a726848fb9dbd25ff4f8c